data/journal_check/bug_refs.json

```diff
@@ -437,5 +437,12 @@
             "sle": ["15-SP4"]
         },
         "type": "ignore"
+    },
+    "Failed to start OpenSSH Daemon": {
```

Comment:
This needs some explanation - where does this happen, why and why can it be ignored?

Reply:
https://bugzilla.opensuse.org/show_bug.cgi?id=1209063